basics.rb

```diff
@@ -8,9 +8,11 @@
 post '/payload' do
   push = JSON.parse(request.body.read)
   puts "I got some JSON: #{push.inspect}"
-  jsontest = "I got some JSON: #{push.inspect}"
-  Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927
-  Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
+
+  Collaborator.add repo_name: push["repository"]["full_name"].to_s, issue_num: push["issue"]["number"].to_i, user_login: push["issue"]["user"]["login"].to_s
+
+  # Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
```

Comment:
Yes, let's leave it as is for now. The instructions currently tell users to go to that issue and comment, but we can delete line 14 once we update the instructions in `training-kit`.